sdk/keyvault/keyvault-admin/samples/v4/javascript/backupRestoreHelloWorld.js

```diff
@@ -35,6 +35,7 @@ async function main() {
   const backupPoller = await client.beginBackup(blobStorageUri, sasToken);
   await backupPoller.pollUntilDone();
 
+  // The folder name should be at the end of the backupFolderUri, as in: ht
```

Comment:
I have the feeling there’s something missing here!

Reply:
Weird... somehow it generated that incorrectly. Good catch!

Reply:
Oh I see what happened, this was somehow removed accidentally in https://github.com/Azure/azure-sdk-for-js/pull/15279/files#diff-8428e954bbb937d85e57627e545d5d4da062359186ba4a62d53bf9793a965899R38 but the samples weren't regenerated so we didn't see it I guess. I can fix that now

Reply:
ohh interesting!!